firmware/Core/Src/telecommands/eps_telecommands.c

```diff
@@ -520,3 +520,82 @@ uint8_t TCMDEXEC_eps_get_piu_housekeeping_data_run_avg_json(
     return 0;
 }
 
+uint8_t TCMDEXEC_eps_get_list_of_enabled_channels(
```

Comment:
Please change this function to give the output as a JSON list-of-strings.

Reply:
Do we want the channels to be the keys or the value for the JSON format?